tests/model_explainability/lm_eval/conftest.py

```diff
@@ -307,3 +280,128 @@ def vllm_emulator_route(
         service=vllm_emulator_service.name,
     ) as route:
         yield route
+
+
+@pytest.fixture(scope="function")
+def lmeval_minio_pvc(
+    admin_client: DynamicClient, model_namespace: Namespace
+) -> Generator[PersistentVolumeClaim, Any, Any]:
+    with PersistentVolumeClaim(
+        client=admin_client,
+        name=f"{MINIO}-pvc",
+        namespace=model_namespace.name,
+        accessmodes=PersistentVolumeClaim.AccessMode.RWO,
+        size="10Gi",
+    ) as pvc:
+        yield pvc
+
+
+@pytest.fixture(scope="function")
+def lmeval_minio_deployment(
```

Comment:
can `tests.model_explainability.guardrails.conftest.minio_llm_deployment` be re-used?
or at least most attributes?